services/apps/search_sync_worker/src/repo/organization.repo.ts

```diff
@@ -40,6 +40,7 @@ export class OrganizationRepository extends RepositoryBase<OrganizationRepositor
             o.address,
             o.attributes,
             o."createdAt",
+            o."manuallyCreated",
```

Comment:
For organizations it only count as an organization if it has members -> check `getOrganizationData` and the `from member_data` and `inner join`.

Reply:
Updated the query, so that it returns organization with a member or `manuallyCreated` is true.